docs/ac_local_roles.rst

```diff
@@ -19,12 +19,12 @@ Configuration::
     [transmogrifier]
     pipeline =
         source
-        ac_local_roles
+        local_roles
 
     ...
 
     [ac_local_roles]
-    blueprint = collective.jsonmigrator.ac_local_roles
+    blueprint = collective.jsonmigrator.local_roles
```

Comment:
Indeed the of the blueprint as defined here:

- https://github.com/collective/collective.jsonmigrator/blob/master/collective/jsonmigrator/blueprints/configure.zcml#L12

is `collective.jsonmigrator.local_roles`, but the example is still broken because you did not rename `[ac_local_roles]` at line 26.
If you fix this I will be glad to merge this PR!